docs/concepts.rst

```diff
@@ -0,0 +1,462 @@
+
+.. _`Concepts`:
+
+Concepts
+========
+
+
+TODO: We need a section on profiles
+
+.. _`The CIM/WBEM Architecture`:
+
+The CIM/WBEM Architecture
```

Comment:
Heading text should be in lower case, except for names:
Architecture -> architecture

This applies to all headings of this new chapter.